### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by Whatabummer** [community/community#67254] [New to GitHub]
I am new to GitHub, honestly I'm new to even know what code is. I'm learning because my phone has been hacked. What I'm having an issue understanding is how can I find the code and get rid of it? Someone please help! I think it's somehow attached either to an app or to a file in my iCloud. I really am not sure how it all works but I think that possibly my cameras have been linked to an outside source prior to hitting my camera app for editing videos purposes then putting the videos on the camera app for my viewing. I think possibly my Facebook has a widget extension connected or plug in allowing voiceover on my phone. I learned this my looking at the analytics on my phone. I think that possibly multiple audio/voice memo recordings I have were messed with, removing the vocals and leaving nothing but static. I think that somehow certain helpful websites are being blocked from my view on safari when I try to find answers and help. I have had my email passwords changed. I've been locked out of own phone 3 weeks ago. Odly, my internet and cameras went down the same night half way across the state from me. Please help!!!

**Comment by mdmzfzl** [comment]
I'm not an expert in cybersecurity, but I can offer some general advice on what you could do if you suspect your phone has been hacked. 

1. Isolate Your Phone:
   - Turn on Airplane Mode or disable Wi-Fi and mobile data.

2. Change Passwords:
   - Change passwords for email, social media, and compromised accounts.
   - Use strong, unique passwords for each account.

3. Scan for Malware:
   - Install a reputable antivirus/anti-malware app.
   - Perform a full scan to detect and remove malicious software.

4. Check for Suspicious Apps:
   - Review installed apps and uninstall unfamiliar or suspicious ones.

5. Update Software:
   - Ensure your phone's OS and apps are up to date.
   - Developers release updates to fix security vulnerabilities.

6. Secure iCloud Account:
   - Change your iCloud password.
   - Enable two-factor authentication for added security.

7. Review App Permissions:
   - Revoke unnecessary or suspicious app permissions.

8. Factory Reset (Last Resort):
   - Backup important data.
   - Perform a factory reset if issues persist.

9. Contact Support:
   - Reach out to your phone manufacturer's or service provider's support.

10. Consider Professional Help:
    - Consult a cybersecurity expert if issues persist.

11. Monitor for Suspicious Activity:
    - Keep an eye on your phone for unusual behavior.
    - Report any problems immediately.

Remember to act quickly and take these steps seriously to mitigate potential damage and improve your overall digital security.

**Comment by queenofcorgis** [comment]
Hi @Whatabummer , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
